CHANGELOG.md

```diff
@@ -32,6 +32,25 @@ Changelog entries are classified using the following labels _(from [keep-a-chang
 
 </details>
 
+## [4.0.8] - 2024-08-22
+
+- backported CVE fix (from v4.0.6) over to 4.x branch
+
+## [4.0.7] - 2024-05-22
+
+- this is basically v4.0.5, with some README updates
+- **it is vulnerable to CVE-2024-4067 & CVE-2024-4068**
+- does NOT break API compatibility
+
+## [4.0.6] - 2024-05-21
+
+- Added `hasBraces` to check if a pattern contains braces.
+- Fixes CVE-2024-4067 & CVE-2024-4068
+- **BREAKS API COMPATIBILITY**
+- Should be labeled as a major release, but it's not.
```

Comment:
@hauserkristof , the top of this PR says

> but fixes https://github.com/advisories/GHSA-952p-6rrq-rcjv & https://github.com/advisories/GHSA-grv7-fg5c-xmjg

IIUC, the second of those and "CVE-2024-4068" is about https://github.com/micromatch/braces , and this PR does not do anything related to that CVE as far as I can see.

The relnote for 4.0.8 should explicitly mention CVE-2024-4067 only?  not CVE-2024-4068 ?

Reply:
True that @jayvdb , I have fixed the title and the description too.

Reply:
The fix for CVE-2024-068 was in:
- v4.0.6 as https://github.com/micromatch/micromatch/commit/92d490dd23da0d02bdc2414ed3929a185a464218
- v4.0.7 on branch v4 as https://github.com/micromatch/micromatch/commit/9cc27cc23826a0e8181db4eebd94ebfca96017a7

Reply:
I dont see a change to the CHANGELOG.md

Reply:
The CHANGELOG.md has been corrected.